doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md

```diff
@@ -1,47 +1,470 @@
+cygnus-ngsi.channels.ch-client-N...
+
+cygnus-ngsi.channels.ch-def.type = ...
+cygnus-ngsi.channels.ch-def...
+```
+
+The related `cygnus_instance_all.conf` should look like:
+
+```
+$ cat /usr/cygnus/conf/cygnus_instance_all.conf
+CYGNUS_USER=cygnus
+CONFIG_FOLDER=/usr/cygnus/conf
+CONFIG_FILE=/usr/cygnus/conf/agent_all.conf
+AGENT_NAME=cygnus-ngsi
+LOGFILE_NAME=cygnus.log
+ADMIN_PORT=8081
+POLLING_INTERVAL=30
+```
+
+Use this kind of advanced configuration in the most restrictive scenarios, where both hardware and exposed ports are very limited and/or highly valuable.
 
 [Top](#top)
 
-##<a name="section4"></a>Single source, multiple storages, parallel sinking
-To be done
+#### <a name="section2.1.4"></a>Orion Context Broker subscriptions
+It seems obvious a per client subscription is required when a per client listener is configured for Cygnus (either by using multiple agents, either by using a single agent with multiple listeners). In that case, each subscription will point to a different endpoint URL.
+
+Nevertheless, it may not seem so obvious how subscriptions look like when a single Cygnus agent with a single listener internally multiplexes the notified context data based on the FIWARE service. In that case, a suscription per client is also required (since Orion does not allow to subscribe to all FIWARE services, due to security reasons), but all subscriptions must point to the same single endpoint URL.
 
 [Top](#top)
 
-##<a name="section5"></a>Multiple sources 
-To be done
+### <a name="section2.2"></a>Scenarios involving multiple FIWARE services
+Another header, the **FIWARE service path**, can be used to segmentate data belonging to a single tenant or client. Such a header is used by Cygnus to prefix the name of tables and collections in database-based sinks, or as a subfolder within the HDFS user space, or as a package/dataset in CKAN, etc.
+
+Analogous to the FIWARE service, the FIWARE service path can be used to parallelize the processing of the context data notified by Orion Context Broker to Cygnus. Either by creating one listener per service-service path pair (then a Cygnus agent can be created per listener, or a single Cygnus agent with multiple listeners can be created), either creating a single listener for all the service-service path pairs and internarlly to Cygnus multiplex the data based on the FIWARE service-path header.
+
+Regarding Orion Context Broker subscriptions, if multiple listeners are used (either multiple agents either a single agent with many listeners) a subscription for each service-service path pair is required, pointing each one to a different endpoint URL. Nevertheless, if a single Cygnus agent with a single listener, internally multiplexing context data based on the FIWARE service path, is used, then a single suscription can be made for all the service paths within the client FIWARE service. This is achieved by using a special FIWARE service path header's value: `/#`.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1420/commits/ea511e5c58c65fc69304692013f33e82ae986f1a